### ISSUE TITLE
JSON-RPC receipts are missing 'v' field

### ISSUE THREAD

**Issue by jimpick** [filecoin-project/ref-fvm#1662]
Using a recent v1.20.0 commit:

```
$ cast tx --rpc-url=https://glif-fvm-v1-20-0-20230213-f427c2b.quick.cluster-3.localnet.farm/rpc/v1 0x8234d78f71b8481fac79f5574cc831ceb8e317f4098546944431410b4d2a8328
Error: 
Deserialization Error: missing field `v` at line 1 column 715. Response: {"chainId":"0x1df5e76","nonce":"0x5","hash":"0x8234d78f71b8481fac79f5574cc831ceb8e317f4098546944431410b4d2a8328","blockHash":"0xbf204c1404e6ffd31a407cf0d78311b7d0347524631171f81c312b127a141593","blockNumber":"0x3f","transactionIndex":"0x5","from":"0x3972e844729522d367bfa1d64368346d7cceea59","to":"0xd6fb0f4cb3ba0387ba9e56113d60ad76f8278b3b","value":"0x0","type":"0x2","input":"0x7a9e5e4b000000000000000000000000d6fb0f4cb3ba0387ba9e56113d60ad76f8278b3b","gas":"0x10000000","maxFeePerGas":"0xb2d1413e","maxPriorityFeePerGas":"0xb2d05e00","accessList":[],"yParity":"0x1","r":"0x63c67b08915b68bc1d5c4dce6862e64d0fef2ff7562dad7dd2ec39776af80ee1","s":"0x214fb4fc8427ae3fbff8f194391efe2ad423f2742d56a0a6c566a603e967086a"}
```

```
$ lotus version
Daemon:  1.19.1-dev+2k+git.f427c2b.dirty+api1.5.0
Local: lotus version 1.19.1-dev+2k+git.f427c2b.dirty
```

This is also breaking `forge script`: https://github.com/filecoin-project/ref-fvm/issues/1641

**Comment by Stebalien** [comment]
Any idea what endpoint that's hitting?

**Comment by Stebalien** [comment]
(nvm)

**Comment by Stebalien** [comment]
I'm very confused:

https://github.com/filecoin-project/lotus/blob/f427c2b566311798063362c437e9187af82ef857/chain/types/ethtypes/eth_transactions.go#L42

**Comment by Stebalien** [comment]
That looks like an upstream bug? The field should be `yParity`, not `v`.

**Comment by Stebalien** [comment]
What version of `cast`?

**Comment by jimpick** [comment]
```
$ cast --version
cast 0.2.0 (250cc85 2022-12-02T00:18:26.200251Z)
```

**Comment by Stebalien** [comment]
Ok, I'm no longer confused. Just annoyed:

https://github.com/filecoin-project/lotus/pull/10102#issuecomment-1428812144

**Comment by Stebalien** [comment]
@jimpick can you test https://github.com/filecoin-project/lotus/pull/10264?

**Comment by Stebalien** [comment]
Should be fixed in the latest release/v1.20.0.
